### ISSUE TITLE
[question] Loading the PPO model after training does seem to load the policy

### ISSUE THREAD

**Issue by Milad-Rakhsha** [hill-a/stable-baselines#1138]
I've read similar questions (e.g. #[30](https://github.com/hill-a/stable-baselines/issues/30)) that were asked here about loading the model after the training but still, I could not figure out what the problem with my model is. My model doesn't seem to be using the trained policy/value networks when I run the following. I am not sure what the problem is with my setup and this does not look like a bug but I was wondering if anyone can tell me if what I am doing is incorrect? (check_env(env) does not give me any warning and the custom env is running fine it just makes random decisions although the in-progress training results looks that the agent has learned the task during training)

```python
log_dir = "./PPO/"
os.makedirs(log_dir, exist_ok=True)

env=Monitor(CustomEnv(8090), log_dir)
# I do have have vecNormalization for the training, and the training is done on a vectorized environment
# but the evaluation is done on a single env 
# env = Normalize(env, norm_obs=True, norm_reward=True,
#                    clip_obs=1.)

check_env(env)
model = PPO.load(log_dir + "/rl_model_8080_12000000_steps", env=env, verbose=True, tensorboard_log=log_dir)
model.set_env(env)#although the above should be able to use the env

mean_reward, std_reward = evaluate_policy(model, env,  n_eval_episodes=10)
print(f"mean_reward:{mean_reward:.2f} +/- {std_reward:.2f}")
```

**System Info**
Describe the characteristic of your environment:
 * conda virtual env, 
 * I have 2080 rtx but don't think it is being used
 * python 3.6.13, stablebaselines 1.1.0, tensorflow 1.14.0, pytorch 1.4.0

**Comment by araffin** [comment]
Hello
answer is in the doc and i highly recommend you to use SB3: https://stable-baselines3.readthedocs.io/en/master/guide/examples.html#pybullet-normalizing-input-features

**Comment by Milad-Rakhsha** [comment]
Thanks for helping out, I think that is my confusion about the vectorized environment. Do I still need to do `VecNormalize.load(...)` even if at the evaluation time the env is not vectorized?
minor correction for future reference, I am using SB3 (stable-baselines3 1.1.0)

**Comment by Milad-Rakhsha** [comment]
Here is how I load the vec_normalize.pkl file along with the model. Looking at the tensorboard results, I can see that `value_loss, entropy_loss, explained_variance, loss, policy_gradient_loss,std, value_loss` start at roughly the same values where I saved the results. However, the `ep_rew_mean` and `ep_len_mean` start from a completely untrained model.

```python
    env=VecMonitor(CustomEnv(8080), log_dir)
    #loading the vec_normalize 
    env = VecNormalize.load(log_dir + filename + "_vec_normalize.pkl", env)
    model = PPO.load(log_dir + filename, verbose=True, tensorboard_log=log_dir, env=env)
```

**Comment by araffin** [comment]
If you can provide a minimal working example to reproduce the issue (without a custom env), please open an issue in SB3 repo.
You should take a look at https://github.com/DLR-RM/stable-baselines3/issues/435 first

**Comment by Milad-Rakhsha** [comment]
Yes, thanks for helping.
